src/utils.js

```diff
@@ -27,6 +27,21 @@ class Utils {
     return newName;
   }
 
+  resolvePath(...values) {
+    for (let i = 0; i < values.length; i++) {
+      let value = values[i];
+
+      if (value) {
+        if (value.path) {
+          value = value.path;
+        }
+
+        return value;
+      }
+    }
+    return false;
```

Comment:
True true 👍

But I remember us having the discussion about returning `false` before. If `resolvePath()` can't find a value, why not just return `null` or `undefined`? That's kinda what `null`/`undefined` are for ;)